linux/Dockerfile

```diff
@@ -9,8 +9,15 @@ RUN sudo curl -sLO https://github.com/PowerShell/PowerShell/releases/download/v7
 
 RUN sudo apt-get update \
   && sudo apt-get install -y \
-    gettext-base \
-  && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Cleaning `/var/lib/apt/lists/*` should be part of the same step as `apt-get update` so as not to leave the data in a layer.